lib/src/firefuel_collection.dart

```diff
@@ -97,11 +96,11 @@ abstract class FirefuelCollection<T extends Serializable>
     return snapshots.map((documentSnapshot) => documentSnapshot.data());
   }
 
-  CollectionReference<Map<String, dynamic>> untypedCollectionRef(
-    FirebaseFirestore instance,
-  ) {
-    return instance.collection(collectionName);
-  }
+  /// Converts a [T] to a [Map<String, Object?>] to upload to Firestore.
```

Comment:
I think because the param is `T? model` the docs should say `[T?]`

```suggestion
  /// Converts a [T?] to a [Map<String, Object?>] to upload to Firestore.
```